webcompat/api/endpoints.py

```diff
@@ -135,33 +120,23 @@ def get_issue_category(issue_category):
         # add "status-" before the filter param to match the naming scheme
         # of the repo labels.
         params['labels'] = 'status-' + issue_category
-        if g.user:
-            issues = github.raw_request('GET', issues_path, params=params)
-        else:
-            issues = proxy_request('get', params=params)
+        return api_request('get', issues_path, params=params)
     elif issue_category == 'closed':
         params['state'] = 'closed'
-        if g.user:
-            issues = github.raw_request('GET', issues_path, params=params)
-        else:
-            issues = proxy_request('get', params=params)
+        return api_request('get', issues_path, params=params)
     # Note that 'new' here is primarily used on the homepage.
     # For paginated results on the /issues page, see /issues/search/new.
     elif issue_category == 'new':
-        if g.user:
-            issues = github.raw_request('GET', issues_path, params=params)
-        else:
-            issues = proxy_request('get', params=params)
+        issues = api_request('get', issues_path, params=params)
         # Do not send random JSON to filter_new
-        if issues.status_code == 200:
-            return (filter_new(json.loads(issues.content)),
-                    issues.status_code, get_headers(issues))
+        # issues is a tuple of format: (content, status_code, response_headers)
+        if issues[1] == 200:
+            return (filter_new(json.loads(issues[0])), issues[1], issues[2])
```

Comment:
The code is becoming less obvious by using simple tuples. 
We could provide either a dict `issues['status_code']` or a namedtuple `issues.status_code`.